internal/connect/api.go

```diff
@@ -8,19 +8,19 @@ import (
 // announceSystem announces a system to SCC
 // https://scc.suse.com/connect/v4/documentation#/subscriptions/post_subscriptions_systems
 // The body parameter is produced by makeSysInfoBody()
-func announceSystem(body []byte) (string, string, error) {
+func announceSystem(body []byte) (string, string, string, error) {
```

Comment:
Is there any path which would return non-empty token from `announceSystem()` and/or `AnnounceSystem()`? If not, then I don't think we need an additional return value. Caller of these functions should take care of this.

Reply:
Good point! Done 👍🏻